tests/particle_attributes/get_RK_traj.py

```diff
@@ -0,0 +1,36 @@
+from scipy import *
```

Comment:
Why is this script here? It seems to be to create the Xvals.txt and Yvals.txt but I'm not sure what they are used for. If these are used, this script needs a comment at the top to say what it does...